contracts/LockRelease.sol

```diff
@@ -69,6 +88,10 @@ contract LockRelease is ILockRelease {
 
             Schedule memory schedule = Schedule(amount, 0);
             schedules[beneficiary] = schedule;
+
+            // @todo: Ensure the following lines are correct
+            _delegate(beneficiary, beneficiary);
+            _moveVotingPower(address(0), beneficiary, amount);
```

Comment:
shouldn't the `_delegate` function above this line be moving the voting power as well?

Reply:
hmm so when `_delegate(beneficiary, beneficary) is called, the `delegatorBalance` in the code snippet below (from `ERC20Votes.sol` will be 0, so the `moveVotingPower` that is called from within that function won't actually move any voting power

![image](https://github.com/decent-dao/dcnt/assets/84364476/081cd421-758a-48ca-9f92-e94894064b0b)

Reply:
ahhh ok ok, yeah makes sense